Plotter.cpp

```diff
@@ -8,9 +8,13 @@
 #include <iomanip>
 #include <iostream>
 
+#include<unistd.h>
```

Comment:
Do you remember why you needed to add this? It isn't available on Windows and also isn't required but I'm guessing some function needs it on Linux?

Reply:
It is not needed, you can remove. I may have added that while debugging